### PR TITLE
Reconsider PR of SortUseNameSpace

### PR DESCRIPTION
**Pull request by piengeng**
Please reconsider the test case I've considered into ```008-inline-html-with-use.*``` sorry for the overlapping of test file, maybe i should separate out next time. Unless In some way OrderUseClause could be used to pass this test case, we'll stick to your version.

related files are https://github.com/piengeng/php.tools/commit/4a8822941594623f7b01f3fec7c998db52fd48c3 and  https://github.com/dericofilho/php.tools/commit/0959e583b5f299f3bc258300256f6137da86e834

### CODE REVIEW

**Comment by dericofilho** [comment]
Please: Remove Laravel. Rebate with Master.

**Comment by dericofilho** [comment]
Please, follow the contribution guidelines and format all codes with KR standard.

**Comment by piengeng** [comment]
erm, what is KR standard? could it be performed on sublime-phpfmt?

**Comment by dericofilho** [comment]
Just use php.tools to format itself. 

# cd src 
# php fmt.src.php Laravel/

**Comment by piengeng** [comment]
did that nothing is changed.

**Comment by dericofilho** [comment]
Ok... No problem. Just rebate with master and remove laravel

**Comment by piengeng** [comment]
I'd tried rebase with upstream(yours) on my local git, then run ```php php.fmt.php Laravel```, not sure what to do next.

**Comment by dericofilho** [comment]
It is still missing removing laravel code

**Comment by piengeng** [comment]
Delete Those extra files not used in your git you mean? I'm terribly sorry, what laravel code? Just seems illogical to delete everything from my fork. I'm sorry as i'm very new to working with git.

**Comment by dericofilho** [comment]
My fault. Don't worry. I meant delete the Laratests, the non related files

File: src/tests-laravel/008-inline-html-with-use.out [hunk 1]
```diff
@@ -1,6 +1,35 @@
...
+use that;
+use power;
+
+<?php
+class twoTwo{
+    function test()
+
+	{
+        use abc;
+        use lmno;
```
Comment (dericofilho) [inline src/tests-laravel/008-inline-html-with-use.out]: I have an issue with this file, in the sense that it is not valid syntax. If you run it through `php -l` it fails. Can you provide a syntax valid test case?

**Comment by piengeng** [comment]
i think that's it,

**Comment by dericofilho** [comment]
Good. I will merge into a branch.

File: src/tests-laravel/008-inline-html-with-use.out (hunk 1 above)
Comment (piengeng) [inline src/tests-laravel/008-inline-html-with-use.out]: let me check

**Comment by piengeng** [comment]
updated some tests to consider alignment, sorted by length then text for the T_USE case, fix a bug in SortUseNameSpace.php due to stupidly re-base my base code !@#$%^ a great lesson learn

**Comment by dericofilho** [comment]
I manually merged and conciliated your PR in the commits https://github.com/dericofilho/php.tools/commit/25e4a60f921eecf55e1d50e1a0c20b4c60cc1b10 https://github.com/dericofilho/php.tools/commit/fa9da508161a75507dc0427dc885dc90875335e3 https://github.com/dericofilho/php.tools/commit/de8e306c330d2af0a151563552e69cec5eb314f5

Could you please check these commits against your large scale laravel tests?

**Comment by piengeng** [comment]
ran on a fresh clone just now, this is with EOL normalized.
laravel/laravel: equals: equals 30, diff 7 -> 81.1%
laravel/framework: equals 182, diff 196 -> 48.1%

Please note that, your OrderSort is missing the secondary sort, which is sort by text, Could you add this as well?
first sort by length of text, then sort by text. then only output.

For the time being, I'll focus on increasing the coverage vs laravel/framework. So this will take some time.

**Comment by dericofilho** [comment]
>Please note that, your OrderSort is missing the secondary sort, which is sort by text, Could you add this as well?
first sort by length of text, then sort by text. then only output.

Done at https://github.com/dericofilho/php.tools/commit/767afc465626c15bc6bffc1dcd699fad6b8fa0cc 

Please, could you check again?

**Pull request closed without merging**